core/src/avm2/globals/flash/media/StageVideoAvailabilityReason.as

```diff
@@ -6,6 +6,7 @@
 package flash.media
 {
 
+    [API("688")]
```

Comment:
I think this should be `670` for Flash Player 10.2:
https://help.adobe.com/en_US/FlashPlatform/reference/actionscript/3/flash/media/StageVideoAvailabilityReason.html